Scheduler/observing_blocks_maintel/BLOCK-399.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "BLOCK-399",
+    "program": "BLOCK-399",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "run_command.py",
+            "standard": true,
+            "parameters": {
+                "component": "MTReflector",
+                "cmd": "open"
+            }
+        },
+        {
+            "name": "maintel/make_lsstcam_calibrations.py",
+            "standard": false,
+            "parameters": {
+                "script_mode": "BIAS_DARK",
+                "n_bias": 3,
+                "n_discard_bias": 1,
+                "n_dark": 3,
+                "n_discard_dark": 1,
+                "exp_times_dark": [15, 30, 38],
+                "wait_between_exposures": 30,
+                "generate_calibrations": true,
+                "do_verify": true
```

Comment:
Should we add a program and reason here? For example

`"program": "$program"`
`"reason": "daily_bias_dark"`